src/transformers/training_args.py

```diff
@@ -1146,6 +1154,14 @@ def __post_init__(self):
             # Go back to the underlying string or we won't be able to instantiate `IntervalStrategy` on it.
             self.evaluation_strategy = self.evaluation_strategy.value
 
+        if self.xpu_backend is not None:
+            warnings.warn(
+                "using `xpu_backend` is deprecated and will be removed in version 5"
```

Comment:
```suggestion
                "using `xpu_backend` is deprecated and will be removed in version 4.31"
```
This is a new addition and experimental. I think we can do a two-minor release deprecation here.